page/specification.php

```diff
@@ -0,0 +1,89 @@
+<?php
+    $id_produk = $_GET['id_produk'];
```

Comment:
Codacy found an issue: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)